src/query/graphite/native/builtin_functions.go

```diff
@@ -1047,6 +1047,51 @@ func logarithm(ctx *common.Context, input singlePathSpec, base int) (ts.SeriesLi
 	return r, nil
 }
 
+// Takes one metric or a wildcard seriesList, and optionally a limit to the number of ‘None’ values
+// to skip over. Continues the line with the last received value when gaps (‘None’ values)
+// appear in your data, rather than breaking your line.
+func interpolate(ctx *common.Context, input singlePathSpec, limit int) (ts.SeriesList, error) {
```

Comment:
nit: can we explicitly call out that it won't interpolate at the end or the start? Graphite isn't very clear on this in their docs but would be useful to have here

Reply:
added